gemrb/GUIScripts/GUISTORE.py

```diff
@@ -1513,35 +1633,38 @@ def SetupItems (pc, Slot, Button, Label, i, type, idx, steal=0):
 					Button.SetState (IE_GUI_BUTTON_SELECTED)
 				else:
 					Button.SetState (IE_GUI_BUTTON_ENABLED)
+			elif Item['Function'] & 4 and not Bag and not Inventory:
+				#containers are always clickable
+				Button.SetState (IE_GUI_BUTTON_ENABLED)
 			else:
 				Button.SetState (IE_GUI_BUTTON_DISABLED)
 
 		if Flags & SHOP_ID:
 			Name = GemRB.GetString (Item['ItemName'])
 			Button.EnableBorder (0, 1)
-			if not steal and type == ITEM_PC:
+			if not steal and type != ITEM_STORE:
 				Price = 1
 		else:
 			Name = GemRB.GetString (Item['ItemNameIdentified'])
 			Button.EnableBorder (0, 0)
 
 		GemRB.SetToken ("ITEMNAME", Name)
-		if Inventory:
+		if Inventory or type == ITEM_STORE and steal:
```

Comment:
Please use parens in all such checks (mixed and+or), so the evaluation order is explicit.

Reply:
Honestly, `and` being of higher precedence than `or` seems super self-evident to me, but I guess I'm not everyone.